Formula/tinyscheme.rb

```diff
@@ -20,35 +22,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -?", 1)
```

Comment:
I would expect the `?` needs to be escaped since `shell_output` uses an intermediate shell.